perception/image_projection_based_fusion/docs/roi-detected-object-fusion.md

```diff
@@ -6,7 +6,10 @@ The `roi_detected_object_fusion` is a package to overwrite labels of detected ob
 
 ## Inner-workings / Algorithms
 
-The detected objects are projected onto image planes, and then if the ROIs of detected objects (3D ROIs) and from a 2D detector (2D ROIs) are overlapped, the labels of detected objects are overwritten with that of 2D ROIs. Intersection over Union (IoU) is used to determine if there are overlaps between them.
+The algorithm is shown below and see `Parameters` for the meaning of each parameter :
+
+1. If `existence_probability` of detected objects is greater than the threshold, they passthrough without doing anything.
```

Comment:
1. If the `existence_probability` of a detected object is greater than the threshold, it is accepted without any further processing.

Reply:
updated at [886ce8a](https://github.com/autowarefoundation/autoware.universe/pull/2178/commits/886ce8a894c29e8fd0df34f56bc743980583a86b).